python/mlc_llm/support/auto_target.py

```diff
@@ -411,6 +412,7 @@ def detect_system_lib_prefix(
         "target": {
             "kind": "opencl",
             "device": "adreno",
+            "max_threads_per_block": 512,
             "host": {
```

Comment:
@tqchen 
We have reverted the generic opencl max thread changes https://github.com/apache/tvm/pull/16972 
We have override the max-thread =512 for android:adreno target.